.scala-steward.conf

```diff
@@ -0,0 +1,4 @@
+updates.pin  = [
+    # the latest version 1.5.5-11 produces a binary conflict with com.gu:thrift-serializer_2.13:5.0.7
+    { groupId = "com.github.luben", artifactId="zstd-jni ", version = "1.5.5-3" }
```

Comment:
There's an extra bit of whitespace in the artificatId which needs to be deleting for the pin logic to work. 
It's my bad, I introduced this earlier

Reply:
no worries, removed now 👍